### ISSUE TITLE
Migrate from Jenkins to Travis

### ISSUE THREAD

**Issue by jmatsushita** [PanicInitiative/PanicButton#25]
This would allow forked projects to be able to run the tests.

**Comment by niccs** [comment]
@Jun,  Please sign in to https://travis-ci.org/ and give access permission to the project.  This is required to configure  the project with Travis build environment.

**Comment by niccs** [comment]
@jmatsushita, yes the build is associated with the correct organization repository. So ideally it is going fine. You can review it and close this issue.

**Comment by jmatsushita** [comment]
Can you please add the Travis build badge on our github's README (http://docs.travis-ci.com/user/status-images/)?

**Comment by niccs** [comment]
Added for dev branch, as Travis is integrated for dev branch.
